polyglot/piranha/test-resources/java/find_and_propagate/configurations/rules.toml

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2022 Uber Technologies, Inc.
+# 
+# <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+# except in compliance with the License. You may obtain a copy of the License at
+# <p>http://www.apache.org/licenses/LICENSE-2.0
+# 
+# <p>Unless required by applicable law or agreed to in writing, software distributed under the
+# License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file contains rules to the specific feature flag API.
+
+#
+# For @stale_flag_name = STALE_FLAG and @treated = true
+# Before 
+#  exp.isToggleEnabled(Experiment.STALE_FLAG)
+# After 
+#  true
+#
+[[rules]]
+name = "find_interface_extension"
+query = """(
+(class_declaration name: (_)@class_name 
+ 				interfaces: (super_interfaces (_) @t) 
```

Comment:
Is this amount of indention normal/expect for toml rules?

Reply:
Actually its the formatting for the tree-sitter query s-expression. I had over-indented there.